modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -71,33 +74,11 @@ public class CatalogUtils {
     /** Max number of distribution zone partitions. */
     public static final int MAX_PARTITION_COUNT = 65_000;
 
-    /**
-     * Default TIMESTAMP type precision: microseconds.
-     *
-     * <p>SQL`16 part 2 section 6.1 syntax rule 36
-     */
-    public static final int DEFAULT_TIMESTAMP_PRECISION = 6;
+    /** Flag indicate no precision applicability. */
+    public static final int PRECISION_NOT_APPLICABLE = Integer.MIN_VALUE;
 
-    /**
-     * Default TIME type precision: seconds.
-     *
-     * <p>SQL`16 part 2 section 6.1 syntax rule 36
-     */
-    public static final int DEFAULT_TIME_PRECISION = 0;
-
-    /**
-     * Default DECIMAL precision is implementation-defined.
-     *
-     * <p>SQL`16 part 2 section 6.1 syntax rule 20
-     */
-    public static final int DEFAULT_DECIMAL_PRECISION = 19;
-
-    /**
-     * Default scale is 0.
-     *
-     * <p>SQL`16 part 2 section 6.1 syntax rule 22
-     */
-    public static final int DEFAULT_SCALE = 0;
+    /** Flag indicate no scale applicability. */
+    public static final int SCALE_NOT_APPLICABLE = Integer.MIN_VALUE;
```

Comment:
why do we need special value for this? I would go with 0, because now we can verify whether precision\scale is applicable by checking ColumnType